pyspecdata/mpl_utils.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 from numpy import r_, pi
+from matplotlib import ticker
```

Comment:
this was just a bug WK and I noticed when we tried to run a processing script we got an error upset about mticker - it didn't know what it was. I'm pretty certain this was a typo and this fix allows us to run scripts as usual